round4/team1.md

```diff
@@ -0,0 +1,98 @@
+# Chrome Extension Idea: PennCourse Extension
+
+## Authors
+
+Eshaan Chichula, Franci Branda-Chen, Jake Garrison Murphy, Matt fu
+
+## Problem Statement
+
+Our Chrome Extension simplifies the course selection process for Penn students by addressing a specific pain point: the need to switch back and forth between Path@Penn and Penn Course Review to assess course difficulty and instructor quality. By integrating this critical information directly into the Path@Penn interface, our tool minimizes the friction involved in course research, allowing students to make informed decisions with ease.
+
+## Target Audience
+
+This extension is tailored for Penn College students who are navigating the course selection process. 
+
+## Description
+
+Aiming to refine the course selection at Penn, our Chrome Extension introduces a user-friendly popup that condenses information from Penn Course Review, such as course difficulty levels and instructor ratings, and integrates it directly onto the Path@Penn page. This enhancement allows students to effortlessly compare and consider courses within a single browser window.
```

Comment:
Very nice and relevant problem to address. Not sure about your data access to achieve this, but as a stretch goal, you could try to index a list of other courses with the same keywords as the course I'm currently viewing to suggest alternatives. What if the course I am looking at is too difficult, has bad ratings, or is full? Could you quickly suggest alternative courses I could look at?

Reply:
We are using [PennCourseReview](https://penncoursereview.com/api/documentation/#tag/PCx-Course/operation/Course%20Search) api's to retrieve course and professor's info

So far, the extension was not thought to make suggestions, but rather display the information about the course/professor that the user wants to know about